Update locale pages for Adventure (#97)
EssentialsX/Website: pages/wiki/Locale.md

# Translations and Custom Messages

> EssentialsX includes messages for a wide range of languages out-of-the-box, as well as extensive support for customising almost every message in the plugin.

Author: All user facing messages will be translatable but debug messages and stack traces aren't guaranteed to be translated

## Changing your server's language

By default, EssentialsX will attempt to use your system language if it is supported by the plugin.  

> You can change the default language by uncommenting and setting the `locale` option in `config.yml`:

Author: This is worded worse

| `lv_LV`  | Lithuanian                      |
| `nl`     | Dutch                           |
| `no`     | Norwegian                       |
| `pl`     | Polish                          |
| `pt`     | Portuguese                      |
| `pt_BR`  | Portuguese (Brazil)             |
| `ro`     | Romanian                        |
| `ru`     | Russian                         |
| `si_LK`  | Sinhala                         |
| `sk`     | Slovak                          |
| `sr_CS`  | Serbian                         |
| `sv`     | Swedish                         |
| `th`     | Thai                            |
| `tr`     | Turkish                         |
| `uk`     | Ukrainian                       |
| `vi`     | Vietnamese                      |
| `zh`     | Simplified Chinese              |
| `zh_HK`  | Traditional Chinese (Hong Kong) |
| `zh_TW`  | Traditional Chinese             |
<!-- | `lol_US` | LOLCAT                          | -->

## Customising messages

In addition to using EssentialsX's built-in translations, you can also customise the text, appearance and functionality of EssentialsX messages!

### Changing the default colours

> **NEW IN 2.21.0:** If you just want to change the colours used in the standard EssentialsX messages, you can do so using the `message-colors` section of `config.yml`:

Reviewer: 🦅
```suggestion
**NEW:** As of EssentialsX 2.21.0, you can change the colors used in the standard EssentialsX messages, just by changing the `message-colors` section of `config.yml`:
```

```yaml

> # Change the default primary and secondary colours used in EssentialsX messages.
> # Some messages may use custom colours, which will need to be edited in the appropriate message files.

Reviewer: 🦅
```suggestion
# Change the default primary and secondary colors used in EssentialsX messages.
# Some messages may use custom colors, which will need to be edited in the appropriate message files.
```

Author: This is the same wording as in the config

# For more information on customising messages, see https://essentialsx.net/wiki/Locale.html
message-colors:
  primary: '#ffaa00'
  secondary: '#ff5555'
```

### Changing the text content

If you want to make more advanced changes to messages, you can create custom message files under the `plugins/Essentials` folder.  
In this directory, create an empty file named `messages_<lang>.properties`, where `<lang>` is the language code for the language you want to edit.  
You can now insert any customised messages in this file, and they will be loaded by EssentialsX the next time you restart your server. For example:

```properties

> TODOInsertExampleHere=<gold>aaaaaaaaa

Reviewer: ```suggestion
broadcast=<primary>Broadcast<secondary>>><white> {0}
```

Author: Already went with another example

```

Note the numbers in curly braces (ie `{0}`, `{1}`). These are template placeholders, which EssentialsX will replace as appropriate before displaying the message. You can compare the messages in the default message files with the appearance in-game to determine which messages are needed.

You can search through the default messages files on [GitHub](https://github.com/EssentialsX/Essentials/tree/2.x/Essentials/src/main/resources). We strongly recommend copying only the messages you want to edit, as this will make it easier to manage and update these files across EssentialsX versions.

**Note: do not edit any messages files directly inside the EssentialsX jar. You will lose these changes when you update the plugin and they will not be converted to the new MiniMessage format automatically.**

### Using rich text elements

> **NEW IN 2.21.0:** EssentialsX now supports the MiniMessage syntax for its messages. This allows you to include gradients, hoverable and clickable elements, custom fonts and more!

Reviewer: ```suggestion
**NEW:** As of 2.21.0, EssentialsX now supports the MiniMessage syntax for its messages. This allows you to include gradients, hoverable and clickable elements, custom fonts, and more!
```
